README.md

```diff
@@ -4,47 +4,30 @@
 [![Documentation](https://docs.rs/tsunami/badge.svg)](https://docs.rs/tsunami/)
 [![Build Status](https://travis-ci.org/jonhoo/tsunami.svg?branch=master)](https://travis-ci.org/jonhoo/tsunami)
 
-`tsunami` provides an interface for running short-lived jobs and experiments on EC2 spot block
+`tsunami` provides an interface for running short-lived jobs and experiments on cloud
 instances. Most interaction with this library happens through
-[`TsunamiBuilder`](struct.TsunamiBuilder.html).
+[`TsunamiBuilder`](struct.TsunamiBuilder.html) and [`Tsunami`](struct.Tsunami.html).
 
-## Examples
+# Example
```

Comment:
In the past, this file has been generated using `cargo readme`. We may want to go away from that, though then I think `README.tpl` should be removed, and we should test the `README` as a doctest using [`doc-comment`](https://docs.rs/doc-comment/0.3.1/doc_comment/).

Reply:
ah, did not know about `cargo readme`. What is `README.tpl`?

Reply:
It's a template for the readme. Basically gives additional "stuff" beyond just what's in the crate-level doc comment in `src/lib.rs` that should be included in the generated `README.md`.

Reply:
I re-generated the README with `cargo readme` - going away from that can be handled separately I think.